storage/local/persistence.go

```diff
@@ -718,6 +715,7 @@ func (p *persistence) loadSeriesMapAndHeads() (sm *seriesMap, err error) {
 					return sm, nil
 				}
 				chunkDescs[i] = newChunkDesc(chunk)
+				chunkDescsTotal-- // Avoid double-counting by newChunkDesc.
```

Comment:
Yeah, the count would be off then. I'll fix (but we would be pretty much toast in those error cases anyway).